package.json

```diff
@@ -14,7 +14,7 @@
     "clean": "rimraf node/dist build",
     "test": "electron-mocha --recursive node/dist/test",
     "lint": "cd node && eslint . --ext .js,.jsx,.ts,.tsx",
-    "format": "p() { prettier ${@:- --write} package.json '*.js' 'node/**/*.{css,js,json,md,scss,ts,tsx}'; }; p"
+    "format": "bash -c \"p() { prettier ${@:- --write} package.json '*.js' 'node/**/*.{css,js,json,md,scss,ts,tsx}'; }; p\""
```

Comment:
I think we can just skip the `format` and `lint` steps on Windows, or even "not on Linux".